dev-games/tiled/metadata.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person" proxied="yes">
+		<email>mechakotik@gmail.com</email>
+		<name>Andrei Sabalenka</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<upstream>
 		<remote-id type="github">bjorn/tiled</remote-id>
```

Comment:
RedirectedUrl: version 1.11.0: metadata.xml: remote-id: permanently redirected: https://github.com/bjorn/tiled -> https://github.com/mapeditor/tiled
ask@bigglane /home/ask/sources/gento

Reply:
Oh, I forgot to correct upstream URL here